packages/react-google-analytics/src/Universal.tsx

```diff
@@ -66,7 +68,7 @@ export default class UniversalGoogleAnalytics extends React.PureComponent<
       allowLinker: true,
     };
 
-    if (debug) {
+    if (debug || disableTracking) {
       // The debug version of the analytics.js library
       // https://developers.google.com/analytics/devguides/collection/analyticsjs/debugging
```

Comment:
Comments probably need to be updated to something like this (with a link to the specific doc heading)
```ts
// Prevent data being sent to Google
// https://developers.google.com/analytics/devguides/collection/analyticsjs/debugging#testing_your_implementation_without_sending_hits
```